mslice/models/projection/powder/mantid_projection_calculator.py

```diff
@@ -33,15 +33,16 @@ def calculate_projection(self, input_workspace_name, axis1, axis2, units):
 
         # can have Q-E or 2theta-E or their transpose.
         if axis1 != DELTA_E_LABEL and axis2 != DELTA_E_LABEL:
-            raise NotImplementedError('Must have an energy axis')
+            raise NotImplementedError("Must have a 'DeltaE' axis")
```

Comment:
Can we use % DELTA_E_LABEL here too? It's all consistent if we change the label string in 1 place then.